src/EventEdition/GuestGrid.vala

```diff
@@ -106,8 +108,16 @@ public class Maya.View.EventEdition.GuestGrid : Gtk.Grid {
                         individual = map_iterator.get_value ();
                         if (individual != null) {
                             try {
-                                individual.avatar.load (32, null);
-                                avatar = new Granite.Widgets.Avatar.from_file (individual.avatar.to_string (), 32);
+                                individual.avatar.load (ICON_SIZE, null);
+                                avatar = new Hdy.Avatar (ICON_SIZE, individual.avatar.to_string (), true);
```

Comment:
This doesn't look right. This would make the named fallback based on the file name. I don't think these two lines are actually necessary at all. We don't need to create a new avatar object here. In the old code this seems to be a workaround for shortcomings of Granite.Avatar

Reply:
Yes, it was based on the old workaround for Granite.Avatar but doesn't makes sense here whilst the suggestion from @tintou does, according to Folks.Individual doc, so I think I'll apply his suggestion.